tutorphilu_extensions/plugin.py

```diff
@@ -173,26 +173,15 @@
 MFE_CONFIG['LOGO_WHITE_URL'] = '{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/static/rg-theme/images/logo.svg'
 MFE_CONFIG['FAVICON_URL'] = '{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/static/rg-theme/images/favicon.ico'
 MFE_CONFIG['SUPPORT_URL_DROPDOWN'] = 'https://edx.readthedocs.io/projects/open-edx-learner-guide/en/open-release-quince.master/SFD_dashboard_profile_SectionHead.html'
+MFE_CONFIG_OVERRIDES = {
```

Comment:
@Lunyachek the whole block should be changed to common settings patch because for now it only changes settings for dev